reco_utils/dataset/python_splitters.py

```diff
@@ -98,17 +103,26 @@ def python_chrono_split(
             col_item=col_item,
         )
 
-    num_of_splits = len(ratio)
-    splits = [pd.DataFrame({})] * num_of_splits
+    # Split by each group and aggregate splits together.
+    splits = []
     df_grouped = data.sort_values(col_timestamp).groupby(split_by_column)
     for name, group in df_grouped:
         group_splits = split_pandas_data_with_ratios(
             df_grouped.get_group(name), ratio, resample=False
         )
```

Comment:
"resample=False" this part is confusion. Is it "re-sampling" or "shuffling"?

This is the codes in split_pandas_data_with_ratios fn:
```
if resample:
        data = data.sample(frac=1, random_state=seed)
```
frac=1 so the intention is to shuffle, not re-sample. Also do we want to reset_index there or keep the original index?

Reply:
addressed in the next commit